doc/dev/image-naming.md

```diff
@@ -0,0 +1,67 @@
+# The Naming Of Docker Images in kubecf.
```

Comment:
This is technically in `cf-operator`, not us (because we don't get to change this).